frontends/concrete-python/tests/tfhers-utils/src/main.rs

```diff
@@ -48,25 +48,55 @@ fn set_server_key_from_file(path: &String) {
     set_server_key(sk);
 }
 
-fn encrypt_with_key(value: u8, client_key: ClientKey, path: &String) {
+fn encrypt_with_key_u8(
+    value: u8,
+    client_key: ClientKey,
+    ciphertext_path: &String,
+) -> Result<(), Box<dyn std::error::Error>> {
     let ct = FheUint8::encrypt(value, &client_key);
-    safe_save(path, &ct)
+
+    safe_save(ciphertext_path, &ct);
```

Comment:
To me doesn't make sense to return always `Ok()` if we want to handle correctly the error here we should not unwrap on safe_save method and return the Result